src/gt4py/next/iterator/transforms/fuse_as_fieldop.py

```diff
@@ -67,6 +67,107 @@ def _is_tuple_expr_of_literals(expr: itir.Expr):
     return isinstance(expr, itir.Literal)
```

Comment:
All changes in this file are merely a small refactoring that extracts the field operator fusion for a single `as_fieldop` from the `FuseAsFieldOp` pass into a dedicated function `fuse_as_fieldop` used by the `InlineDynamicShifts` pass.